packages/api/src/index.ts

```diff
@@ -1,10 +1,23 @@
+import { getHyperDXMetricReader } from '@hyperdx/node-opentelemetry/build/src/metrics';
+import { HostMetrics } from '@opentelemetry/host-metrics';
+import { MeterProvider, MetricReader } from '@opentelemetry/sdk-metrics';
 import { serializeError } from 'serialize-error';
 
 import * as config from '@/config';
 import Server from '@/server';
 import { isOperationalError } from '@/utils/errors';
 import logger from '@/utils/logger';
 
+if (config.IS_DEV) {
+  // Start collecting host metrics
+  const meterProvider = new MeterProvider({
```

Comment:
this doesn't seem to attach any useful identifying resource attributes either, is it because we're not using the `NodeSDK` for metrics?

Reply:
good point. it doesnt speak to the sdk ecosystem. [the implementation](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/37d8ce16faf4d3d3b45eaa0f7bdeddcfa47f139d/packages/opentelemetry-host-metrics/src/metric.ts) is very weird now I look at it. we reuse the metric reader instead of plugging it into sdk and that's why resource detector doesn't come into play